solr/solr-ref-guide/src/parallel-sql-interface.adoc

```diff
@@ -240,16 +240,20 @@ The parallel SQL interface supports and pushes down most common SQL operators, s
 |= |Equals |`fielda = 10` |`fielda:10`
 |<> |Does not equal |`fielda <> 10` |`-fielda:10`
 |!= |Does not equal |`fielda != 10` |`-fielda:10`
-|> |Greater than |`fielda > 10` | `fielda:{10 TO *]`
-|>= |Greater than or equals |`fielda >= 10` | `fielda:[10 TO *]`
-|< |Less than |`fielda < 10` | `fielda:[* TO 10}`
-|\<= |Less than or equals |`fielda \<= 10` | `fielda:[* TO 10]`
+|> |Greater than |`fielda > 10` |`fielda:{10 TO *]`
+|>= |Greater than or equals |`fielda >= 10` |`fielda:[10 TO *]`
+|< |Less than |`fielda < 10` |`fielda:[* TO 10}`
+|\<= |Less than or equals |`fielda \<= 10` |`fielda:[* TO 10]`
+|IN |Specify multiple values (shorthand for multiple OR clasues) |`fielda IN (10,20,30)` |`(fielda:10 OR fielda:20 OR fielda:30)`
+|LIKE |Wildcard match on string or text fields |`fielda LIKE 'day%'` |`{!complexphrase}fielda:"day*"`
+|BETWEEN |Range match |`fielda BETWEEN 2 AND 2` |`((fielda: [2 TO *]) AND (fielda: [* TO 4]))`
```

Comment:
Did you mean
```suggestion
|BETWEEN |Range match |`fielda BETWEEN 2 AND 4` |`((fielda: [2 TO *]) AND (fielda: [* TO 4]))`
```

Reply:
Also, it's weird to me that BETWEEN needs to be a conjunction of two ranges?

Reply:
Agreed on that ... that's existing code but will look into changing it ;-)

Reply:
heh ... it's because Calcite transforms the expression into `AND(>=($2, 2), <=($2, 4))`, i.e. BETWEEN is just syntactic sugar and the Solr code ends up seeing: `field >= 2 AND field <= 4`.